src/CodeGen_Vulkan_Dev.cpp

```diff
@@ -2110,8 +2117,10 @@ void CodeGen_Vulkan_Dev::SPIRV_Emitter::visit(const Shuffle *op) {
                     SpvId type_id = builder.declare_type(op->type);
```

Comment:
The comment at line 2102 is outdated/wrong. Shuffles do not support "don't care", at least not according to:
https://github.com/halide/Halide/blob/65b08efa78baf964387b9615b4c80e43d4cb967a/src/IR.cpp#L774-L776
Unless you internally, during Vulkan lowering/codegen, are using -1 as a don't care value, which also looks like a code smell.